### PR TITLE
fix release npm

### PR DESCRIPTION
**Pull request by Sotatek-TinnNguyen**
## Proposed changes

- Describe your changes to communicate to the maintainers why we should accept this pull request.
- If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or Enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [ ] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn-developer-sdk/blob/main/CONTRIBUTING.md) doc
- [ ] I have signed the CLA by posting "I have read the CLA Document and I hereby sign the CLA" in the comment
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you proposed and what alternatives you have considered, etc.

### CODE REVIEW

**Review by toniya-klaytn — APPROVED** [review]

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -26,6 +26,7 @@ jobs:
       # - name: testing
       #   run: npm run test
       - name: Publish package on NPM 📦
-        run: npx pnpm --filter "@klaytn-developer-sdk/*" -r publish --publish-branch main --no-git-check
-        env:
-          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+        uses: JS-DevTools/npm-publish@v1
```
Comment (praveen-klaytn) [inline .github/workflows/publish.yml:29]: Hi @Sotatek-TinnNguyen , Publish should perform below things.
1) It should publish only the sub packages (kds-cli, oracle, celer, wormhole)  and avoid publishing the root package.
2) It should only publish the sub package which package.json's version got changed. Ex: if only oracle subpackage version is changed, on that needs be published.

May i know if this performs as mentioned above ?

Comment (praveen-klaytn) [reply .github/workflows/publish.yml:29]: So when we run the publish.yml below packages should get published. It should avoid publishing root package.
@klaytn-developer-sdk/kds-cli   -> path=/packages/kds-cli
@klaytn-developer-sdk/oracles   -> path=/packages/oracles-starter-kit
@klaytn-developer-sdk/bridges-wormhole   -> path=/packages/bridges-starter-kit/wormhole
@klaytn-developer-sdk/bridges-celer   -> path=/packages/bridges-starter-kit/celer
@klaytn-developer-sdk/dexs   -> path=/packages/dexs-starter-kit

File: package.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
   "repository": "https://github.com/klaytn/klaytn-developer-sdk",
   "author": "Klaytn Community <developer@klaytn.foundation>",
   "license": "MIT",
-  "private": true,
+  "private": false,
```
Comment (praveen-klaytn) [inline package.json:7]: We can ignore this change as well, as the root package is not published.

**Comment by Sotatek-TinnNguyen** [comment]
Issue will be resolved by changing org to `@klaytn` as discussed. Praveen will do it with fixing lint

**Pull request closed without merging**